constants/testconstants/testconstants.go

```diff
@@ -0,0 +1,11 @@
+package testconstants
+
+import "time"
+
+const (
+	DefaultNetworkID   = uint32(1337)
+	DefaultNetworkSize = 5
```

Comment:
These should be in a `_test.go` file since they're only used in tests. 

```
StopTimeout        = 10 * time.Second
RemoveTimeout      = 30 * time.Second
PollTimeout        = 5 * time.Second
```
^ I'd prefer these go in the `k8s` package since they're only used there. Seems reasonable to use different timeouts for local runner and k8s runner